std_msgs/Header header
ObstacleDelphiSRR3[] obstacles